etcdmgmt/etcd-mgmt.go

```diff
@@ -0,0 +1,73 @@
+package etcdmgmt
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+func StartEtcd() error {
+
+	log.Info("Starting etcd")
+	hostname, err := os.Hostname()
+	if err != nil {
+		log.Fatal("Could not able to get hostname")
+		return err
+	}
+
+	listenClientUrls := "http://" + hostname + ":2379"
+
+	advClientUrls := "http://" + hostname + ":2380"
+
+	etcdStart := exec.Command("/bin/etcd",
```

Comment:
we are starting etcd it self. If we are executing any etcd command then we should have etcdCommand here.